src/updates.js

```diff
@@ -95,16 +95,33 @@ class Updates {
   }
 
   async handleUpdate(res, account, repository, platform, version) {
-    const latest = await this.cachedGetLatest(
+    let latest = await this.cachedGetLatest(
       account,
       repository,
       platform,
       version
     );
 
+    if (platform.includes(PLATFORM.DARWIN)) {
+      const latestUniversal = await this.cachedGetLatest(
+        account,
+        repository,
+        PLATFORM_ARCH.DARWIN_UNIVERSAL,
+        version
+      );
+
+      if (
+        latestUniversal &&
+        semver.gt(latestUniversal.version, latest.version)
```

Comment:
@MarshallOfSound - moved the semver check here to be part of the universal replacement logic